controllers/keystoneapi_controller.go

```diff
@@ -411,9 +428,6 @@ func (r *KeystoneAPIReconciler) reconcileInit(
 	}
 	if bootstrapjob.HasChanged() {
 		instance.Status.Hash[keystonev1.BootstrapHash] = bootstrapjob.GetHash()
-		if err := r.Client.Status().Update(ctx, instance); err != nil {
-			return ctrl.Result{}, err
-		}
 		r.Log.Info(fmt.Sprintf("Job %s hash added - %s", jobDef.Name, instance.Status.Hash[keystonev1.BootstrapHash]))
```

Comment:
as you removed an immediate update, this message becomes no longer relevant - nothing has been added by far?